mlflow/__init__.py

```diff
@@ -66,14 +66,13 @@
 
 _configure_mlflow_loggers(root_module_name=__name__)
 
-if sys.version_info.major == 2:
+if sys.version_info[:2] == (3, 5):
     warnings.warn(
-        "MLflow support for Python 2 is deprecated and will be dropped in a future "
```

Comment:
Should we determine the version to drop Python 3.5 and mention that version in the warning message?